### PR TITLE
Ensure all commands are listed in the README

### PR DESCRIPTION
**Pull request by danielbachhuber**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,29 +180,560 @@ wp plugin
...
+		Render output in a particular format.
+		---
+		default: table
+		options:
+		  - table
+		  - csv
+		  - json
+		  - yaml
+		---
+
```
Comment (gitlost) [inline README.md]: Again maybe an AVAILABLE FIELDS section referring down to **wp theme list**.

File: README.md [hunk 2]
```diff
@@ -80,38 +743,419 @@ wp theme
...
+    Installing Twenty Sixteen (1.2)
+    Downloading install package from http://downloads.wordpress.org/theme/twentysixteen.1.2.zip...
+    Unpacking the package...
+    Installing the theme...
+    Theme installed successfully.
+    Activating 'twentysixteen'...
+    Success: Switched to 'Twenty Sixteen' theme.
+
+    # Install from a local zip file
+    $ wp theme install ../my-theme.zip
```
Comment (gitlost) [inline README.md]: Missing output.
```
Unpacking the package...
Installing the theme...
Theme installed successfully.
Success: Installed 1 of 1 themes.
```

File: README.md [hunk 3]
```diff
@@ -80,38 +743,419 @@ wp theme
...
+
+    # Install from a remote zip file
+    $ wp theme install http://s3.amazonaws.com/bucketname/my-theme.zip?AWSAccessKeyId=123&Expires=456&Signature=abcdef
```
Comment (gitlost) [inline README.md]: Missing output, same as above I think?

File: README.md [hunk 4]
```diff
@@ -80,38 +743,419 @@ wp theme
...
+
+**AVAILABLE FIELDS**
+
+These fields will be displayed by default for each theme:
+
+* name
+* status
+* update
+* version
+
```
Comment (gitlost) [inline README.md]: Mention that the status field can have values "active", "inactive", "parent".
Mention that the update field can have values "available", "none".

File: README.md [hunk 5]
```diff
@@ -80,38 +743,419 @@ wp theme
...
+    $ wp theme list --status=inactive --format=csv
+    name,status,update,version
+    twentyfourteen,inactive,none,1.7
+    twentysixteen,inactive,available,1.1
+
+
+
+### wp theme mod get
+
+Get one or more theme mods.
```
Comment (gitlost) [inline README.md]: Maybe do "mods (modifications)" on first use?

File: README.md [hunk 6]
```diff
@@ -80,38 +743,419 @@ wp theme
...
+	[--per-page=<per-page>]
+		Optional number of results to display. Defaults to 10.
+
+	[--field=<field>]
+		Prints the value of a single field for each theme.
+
+	[--fields=<fields>]
+		Ask for specific fields from the API. Defaults to name,slug,author,rating. Acceptable values:
+
+    **name**: Theme Name
```
Comment (gitlost) [inline README.md]: As above, Markdown treating this as a code block.

File: README.md [hunk 7]
```diff
@@ -80,38 +743,419 @@ wp theme
...
+    +---------------+-------------+-------------+---------+
+    | name          | old_version | new_version | status  |
+    +---------------+-------------+-------------+---------+
+    | twentyfifteen | 1.4         | 1.5         | Updated |
+    | twentysixteen | 1.1         | 1.2         | Updated |
+    +---------------+-------------+-------------+---------+
+    Success: Updated 2 of 2 themes.
+
+    # Update all themes
+    $ wp theme update --all
```
Comment (gitlost) [inline README.md]: Missing output, eg
```
Enabling Maintenance mode...
Downloading update from https://downloads.wordpress.org/theme/flat.1.7.8.zip...
Unpacking the update...
Installing the latest version...
Removing the old version of the theme...
Theme updated successfully.
Disabling Maintenance mode...
+------+-------------+-------------+---------+
| name | old_version | new_version | status  |
+------+-------------+-------------+---------+
| flat | 1.7.7       | 1.7.8       | Updated |
+------+-------------+-------------+---------+
Success: Updated 1 of 1 themes.
```

**Comment by danielbachhuber** [comment]
@gitlost In the interest of having a README for the command, I'm going to punt on your suggestions for now.

The README is automatically generated from the command docs, so we can make incremental improvements to the command docs, and then they'll be reflected in the README.

**Pull request merged**